qtconsole/base_frontend_mixin.py

```diff
@@ -62,6 +63,23 @@ def kernel_client(self, kernel_client):
         # we connected.
         if kernel_client.channels_running:
             self._started_channels()
+    
+    def _make_blocking_client(self):
+        kc = self.kernel_client
+        if kc is None:
+            return
+        from jupyter_client import BlockingKernelClient
```

Comment:
Should this be a regular import rather than a local one? jupyter_client is a plain dependency.